EthOn.rdf

```diff
@@ -1904,18 +1904,18 @@
         </owl:annotatedTarget>
         <rdfs:comment>All create transactions are directed toward a Contract Account they create.</rdfs:comment>
     </owl:Axiom>
-
+    
 
 
     <!-- http://ethon.consensys.net/EthOnConcept -->
 
     <owl:Class rdf:about="http://ethon.consensys.net/EthOnConcept">
         <suggestedStringRepresentation xml:lang="en">EthonConcept</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">Groups al EthOn concepts.</rdfs:comment>
+        <rdfs:comment xml:lang="en">The purpose of concepts in EthOn is to group a finite number of concrete instances of states and/or relationships between states under a single indefinitely bounded term.</rdfs:comment>
```

Comment:
Hmm, this seems awkwardly phrased. My suggestion is to keep this more simple, like for example: "This is the root concept of all concepts that are original to the EthOn OWL ontology."